weather-radar-gateway/routes/index.js

```diff
@@ -0,0 +1,9 @@
+const express = require('express')
+const router = express.Router()
+
+router.all('/:apiName', (req, res) => {
```

Comment:
Add comment explaining working of the service.

Reply:
I have added comments. Please review again.

Reply:
Its looks good. Merging